### ISSUE TITLE
Can't select sibling options

### ISSUE THREAD

**Issue by sntran** [philss/floki#39]
Test case:

```elixir
iex> html = "<select><option value=\"foo\">Foo</option><option value=\"bar\">Bar</option><option value=\"barz\">Barz</option></select>"
iex> Floki.parse(html) |> Floki.find("select option + option")
** (ArgumentError) argument error
    :erlang.hd([])
    lib/floki/finder.ex:90: Floki.Finder.traverse_sibling/4
    lib/floki/finder.ex:60: Floki.Finder.traverse/4
    lib/floki/finder.ex:40: Floki.Finder.traverse/4
    lib/floki/finder.ex:56: Floki.Finder.traverse/4
    lib/floki/finder.ex:44: Floki.Finder.traverse/4
    lib/floki/finder.ex:23: Floki.Finder.find/2
```

Floki version: 0.6.1

**Comment by philss** [comment]
Hi @sntran! Thank you for the report!

It is fixed in the version 0.7.0. Please check that.

**Comment by sntran** [comment]
:clap:  Thank you!
